src/components/dashboard/DashboardContainer.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { PureComponent } from 'react';
```

Comment:
@blackshady I noticed you used `PureComponent` instead of `Component`, is there a particular reason?

Reply:
PureComponent is exactly the same as Component except that it handles the shouldComponentUpdate method for you.